test.js

```diff
@@ -141,20 +135,69 @@ describe('cli', function () {
       // mock git by throwing on attempt to commit
       return mockGit('console.error("tag, you\'re it"); process.exit(128);', 'tag')
         .then(function (unmock) {
-          writePackageJson('1.0.0')
-
           var result = execCli()
           result.code.should.equal(1)
-          result.stdout.should.match(/tag, you're it/)
+          result.stderr.should.match(/tag, you're it/)
 
           unmock()
         })
     })
   })
 
-  it('handles commit messages longer than 80 characters', function () {
-    writePackageJson('1.0.0')
+  describe('with mocked conventionalRecommendedBump', function () {
+    beforeEach(function () {
+      mockery.enable({warnOnUnregistered: false, useCleanCache: true})
+      mockery.registerMock('conventional-recommended-bump', function (_, cb) {
+        cb(new Error('bump err'))
+      })
+    })
+
+    afterEach(function () {
+      mockery.deregisterMock('conventional-recommended-bump')
```

Comment:
Yeah I've been needing something like `mockery` for a while, and I cant believe I missed it before when I was looking for it.. It's really cool.